utils/rpms/daos.spec

```
@@ -4,9 +4,9 @@
 
 %if (0%{?suse_version} >= 1500)
 # until we get an updated mercury build on 15.2
-%global mercury_version 2.0.0~rc1-1.suse.lp151
+%global mercury_version 2.0.0~rc2-1.suse.lp152
 %else
-%global mercury_version 2.0.0~rc1-1%{?dist}
+%global mercury_version 2.0.0~rc2-1%{?dist}
 %endif
```

We should have this by now and can get rid of the leap branch.  Hopefully before we ever need it again, we get on a more normal _mercury_ consumer train.

```suggestion
%global mercury_version 2.0.0~rc2-1%{?dist}
```